src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -174,6 +174,33 @@ private function loadCacheControl(ContainerBuilder $container, array $config)
 
     private function parseRuleMatcher(ContainerBuilder $container, array $match)
     {
+        $requestMatcher = $this->parseRequestMatcher($container, $match);
+        $responseMatcher = $this->parseResponseMatcher($container, $match);
+
+        $signature = serialize([(string) $requestMatcher, (string) $responseMatcher]);
+        $id = $this->getAlias().'.cache_control.rule_matcher.'.md5($signature);
+
+        if (!$container->hasDefinition($id)) {
```

Comment:
I don't see the point of this. This can't be defned. And if it was there is an error